rdkit/Chem/UnitTestSuppliers.py

```diff
@@ -1,104 +1,254 @@
-# $Id$
 #
-#  Copyright (C) 2003-2006  Rational Discovery LLC
+#  Copyright (C) 2002-2006  greg Landrum and Rational Discovery LLC
```

Comment:
Ah, now I think I see what happened. The contents of this file are now wrong. UnitTestSuppliers.py somehow got overwritten with UnitTestSurf.py